bokehjs/src/lib/models/tools/inspectors/hover_tool.ts

```diff
@@ -429,6 +432,7 @@ export namespace HoverTool {
     renderers: p.Property<RendererSpec>
     names: p.Property<string[]>
     mode: p.Property<HoverMode>
+    ignore_muted: p.Property<boolean>
```

Comment:
Maybe let's have a muted policy? This way we could allow in future to e.g. show muted entries with "muted" (e.g. faded) styling or deprioritize such entries (list them after non-muted entries), etc. After all, muted glyph != hidden glyph.

Reply:
I've thought about it for some time but couldn't come up with a good argument that would tip over the balance, to decided to go with the simplest option.
I guess your voice tips the balance the other way. :) I'll update it.